octavia_proxy/api/v2/controllers/l7rule.py

```diff
@@ -39,8 +40,13 @@ def __init__(self, l7policy_id):
                          [wtypes.text], ignore_extra_args=True)
     def get_one(self, id, fields=None):
         """Gets a single l7rule's details."""
+        pcontext = pecan_request.context
         context = pecan_request.context.get('octavia_context')
-        l7rule = self.find_l7rule(context, self.l7policy_id, id)
+        query_params = pcontext.get(constants.PAGINATION_HELPER).params
+        is_parallel = query_params.pop('is_parallel', False)
```

Comment:
is it really possible we ever get this QP (same question valid everywhere)?

Reply:
only during direct call to api, i suppose, but in next pr, i will enable parallel execution by default.